Tasks/FuncToolsInstallerV0/src/utils.ts

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as toolLib from 'azure-pipelines-tool-lib/tool';
+import * as functoolsutility from "func-tools-common/functoolsutility";
+
+export async function getFuncToolsVersion(): Promise<string> {
+    const version = tl.getInput("version");
+    if (version && version != "latest") {
+        return sanitizeVersionString(version);
+    }
+
+    console.log(tl.loc("FindingLatestFuncToolsVersion"));
+    const latestVersion =  await functoolsutility.getLatestFuncToolsVersion();
+    console.log(tl.loc("LatestFuncToolsVersion", latestVersion));
+    return latestVersion;
+}
+
+export async function downloadFuncTools(version: string): Promise<string> {
+    return await functoolsutility.downloadFuncTools(version);
+}
+
+// handle user input scenerios
+function sanitizeVersionString(inputVersion: string) : string{
+    const version = toolLib.cleanVersion(inputVersion);
```

Comment:
Would you like to add support to versions like `v1` or `1.*` or something like that later down the line?

Reply:
Not planned as of now. I don't see a way we can achieve that.